ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -75,16 +79,51 @@ images:
       paths:
       - destination_dir: .
         source_path: /usr/bin/oc
-    openshift_release_golang-1.17:
+    openshift_release_golang-1.18:
       as:
-      - registry.ci.openshift.org/openshift/release:golang-1.17
+      - registry.ci.openshift.org/openshift/release:golang-1.18
   to: capk-oc-bin-image
+- build_args:
+  - name: ARCH
+    value: amd64
+  - name: ldflags
+  dockerfile_literal: |-
+    FROM src
+    COPY oc hypershift /usr/bin/
+    RUN mkdir -p /go/src/github.com/openshift && \
+        cd /go/src/github.com/openshift && \
+        git clone https://github.com/openshift/hypershift.git && \
+        cd hypershift/ && \
+        go mod tidy && go mod vendor && \
+        make e2e
+    WORKDIR /go/src/github.com/openshift/hypershift
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    hypershift-operator-4.11:
```

Comment:
Not critical
This will have to be maintained all the time (change versions)
Isn't there any way to get the latest release?

Reply:
It would have to be handled manually

Reply:
https://quay.io/repository/hypershift/hypershift-operator?tab=tags there's a `latest` tag. I don't know if that helps us here though or is even the right thing to do. It's possible we should be explicitly picking the right operator release tied to an ocp version.